packages/roosterjs-editor-core/lib/corePlugins/DOMEventPlugin.ts

```diff
@@ -144,6 +150,7 @@ export default class DOMEventPlugin implements PluginWithState<DOMEventPluginSta
 
     private onFocus = () => {
         this.editor.select(this.state.selectionRange);
+        setSelectedTableCells(this.editor);
```

Comment:
Normally, code from roosterjs-editor-dom package should not know anything about editor.

This looks should be something done by a core plugin, but not a DOM level API